packages/buildbucket-dart/lib/buildbucket_proto.dart

```diff
@@ -0,0 +1,8 @@
+// Copyright 2020 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+library buildbucket_proto;
+
+export 'src/generated/go.chromium.org/luci/buildbucket/proto/build.pb.dart' show Build;
+
```

Comment:
Do you plan to add a `buildbucket_dart` library later?

Reply:
Maybe, it will definitely simplify app_dart buildbucket implementation but it will also require a migration to buildbucket v2(currently using v1 everywhere).